libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -920,7 +920,7 @@ button.gd-button-primary.gd-upsell-button {
     .gd-icon-star {
         width: 14px;
         height: 13px;
-        margin: auto auto 6.5px 11px;
+        margin: auto 8px 6.5px 11px;
```

Comment:
Just a confirmation: do we really need to change this?

Reply:
the icon will not margin with text when having a long text, have tested and found it 
<img width="1584" alt="Screen Shot 2022-10-17 at 10 44 40" src="https://user-images.githubusercontent.com/58030757/196085031-7dc51c71-0f84-4bc6-99d1-dadad8f86281.png">